packages/frontend-2/server/api/v2/healthz/readiness.get.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  ApolloClient,
+  ApolloQueryResult,
+  gql,
+  HttpLink,
+  InMemoryCache,
+  NormalizedCacheObject
+} from '@apollo/client/core'
+
+type GraphQLClient = ApolloClient<NormalizedCacheObject>
+
+const serverInfoVersionQuery = gql`
```

Comment:
Use `graphql()` out of `~~/lib/common/generated/gql` instead of `gql()`, this will ensure TypeScript types are auto-generated (when you run the dev server through `yarn dev`, or separately through `yarn gqlgen`) and you'll get properly typed result structures. There's various examples of such queries/mutations across the `frontend-2` package also.